### DISCUSSION TITLE
Welcome to Monocle Discussions!

### DISCUSSION THREAD

**Discussion by morucci** [change-metrics/monocle#1038] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by mwpierce63** [comment]
I need to completely uninstall the monocle package from my workstation but I can't see how to do that, can someone help?

**Comment by morucci** [comment]
Hi, you can run `docker compose rm`.
